Bottom: 8056b5c93aa56b0526e544bdbe39a58171bb395b
Top:    ffb403edd2d216a9cf84547e374cd1415d9311dc
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-27 19:19:44 +0000

Refresh of find-c-avoid-modifying-bounds-

---

diff --git a/src/find.c b/src/find.c
index be88bab..1b99ba4 100644
--- a/src/find.c
+++ b/src/find.c
@@ -785,8 +785,8 @@ LookupLOConnectionsToPVList (bool AndRats)
       if (AndRats)
         {
           if (setjmp (info.env) == 0)
-            r_search (PCB->Data->rat_tree, &search_box,
-                      NULL, LOCtoPVrat_callback, &info);
+            r_search (PCB->Data->rat_tree, &search_box, NULL,
+                      LOCtoPVrat_callback, &info);
           else
             return true;
         }
